Mobile/starter_project_matador/lib/features/example/presentation/screen/signup.dart

```diff
@@ -0,0 +1,216 @@
+// Import flutter heplper library
+import 'package:flutter/material.dart';
+
+class SignUp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+        title: 'Sign Up Page',
+        home: Scaffold(
+            backgroundColor: Colors.white,
+            body: SafeArea(
+                child: ListView(children: <Widget>[
+              SizedBox(
+                height: 130,
+                width: 234,
+                child: Image(
+                  height: 54,
+                  width: 141,
+                  image: NetworkImage(
+                      "https://media.licdn.com/dms/image/D4E16AQGS-26jpTdk2w/profile-displaybackgroundimage-shrink_200_800/0/1671689947030?e=2147483647&v=beta&t=6y-EPCeib7vMuOKrklKkiVnqCw8KEH0q6VfJovmBb9A"),
+                ),
+              ),
+              Container(
+                height: 76,
+                width: 295,
+                decoration: BoxDecoration(
+                  color: Color(0xff376AED),
+                  borderRadius: BorderRadius.only(
+                    topLeft: Radius.circular(28),
+                    topRight: Radius.circular(28),
+                  ),
+                  boxShadow: [
+                    BoxShadow(
```

Comment:
You can add a colors file in the utils folder to add all your colors there and then import the file in this file to access the colors.